packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -273,3 +273,84 @@ class VirtualServiceBackend extends Backend {
     };
   }
 }
+
+/**
+ * Generated Connection pool config
+ */
+export interface ConnectionPoolConfig {
```

Comment:
Let's not put this here. Let's actually put this in a completely separate file in the `lib/private` subdirectory, where the relevant code will be able to import it from. Of course, do not add this new file in `lib/private` to the exports of the package in `index.ts`.

This way, no linter exclusions will be needed, and this type won't be needlessly exposed in the public API of this module.

Reply:
perfect! I did not know that about the `lib/private` folder